packages/litexa/src/command-line/templates/inlined/typescript/package.json

```diff
@@ -6,8 +6,8 @@
     "compile": "npx tsc",
     "compile:watch": "npx tsc -w",
     "deploy": "npm run compile && litexa deploy",
-    "test": "npm run compile && litexa test",
-    "test:watch": "npm run compile:watch | litexa test -w"
+    "test:litexa": "npm run compile && litexa test",
+    "test:litexa:watch": "npm run compile:watch | litexa test -w"
```

Comment:
Could alternatively update the README to reflect the correct NPM script to run for testing - might be more convenient than forcing the additional tag in the script name?

Reply:
I thought I'd do it this way because (a) these scripts run Litexa tests and not TypeScript tests,  (b) the README.md describes these scripts as Litexa test scripts, and (c) they align with the Litexa test scripts in the other templates.